docker-compose.yml

```diff
@@ -11,38 +11,51 @@ services:
       - jobbergate-net
     volumes:
       - ./jobbergate-api/jobbergate_api:/app/jobbergate_api
-    command: uvicorn jobbergate_api.main:app --reload --workers 1 --host 0.0.0.0 --port 8000
+    command: poetry run jobbergate dev-server --port=8000
     environment:
-      - DATABASE_HOST=${NEXTGEN_DATABASE_HOST:-nextgen-db}
-      - DATABASE_USER=${NEXTGEN_DATABASE_USER:-nextgen}
-      - DATABASE_PSWD=${NEXTGEN_DATABASE_PSWD:-nextgen}
-      - DATABASE_NAME=${NEXTGEN_DATABASE_NAME:-nextgen}
-      - DATABASE_PORT=${NEXTGEN_DATABASE_PORT:-5432}
-      - S3_ENDPOINT_URL=${S3_ENDPOINT_URL:-http://s3:9000}
+      - DATABASE_HOST=${DATABASE_HOST:-db}
+      - DATABASE_USER=${DATABASE_USER:-omnivector}
+      - DATABASE_PSWD=${DATABASE_PSWD:-local-pswd}
+      - DATABASE_NAME=${DATABASE_NAME:-jobbergate}
+      - DATABASE_PORT=${DATABASE_PORT:-5432}
+      - S3_ENDPOINT_URL=${S3_ENDPOINT_URL:-http://minio:9000}
+      - S3_BUCKET_NAME=${S3_ENDPOINT_URL:-jobbergate-resources}
       - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-FAKE_ACCESS_KEY_ID}
       - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-FAKE_SECRET_ACCESS_KEY}
-      - ARMASEC_DOMAIN=${ARMASEC_DOMAIN}
-      - ARMASEC_AUDIENCE=${ARMASEC_AUDIENCE}
+      - ARMASEC_DOMAIN=${AUTH0_DOMAIN}
+      - ARMASEC_AUDIENCE=${AUTH0_AUDIENCE}
       - ARMASEC_DEBUG=${ARMASEC_DEBUG}
     ports:
       - 8000:8000
-    depends_on:
-      - nextgen-db
 
-  nextgen-db:
+  db:
     image: postgres
     restart: always
     networks:
       - jobbergate-net
     volumes:
       - postgres_data:/var/lib/postgresql/data/
     environment:
-      - POSTGRES_PASSWORD=${NEXTGEN_DATABASE_PSWD:-nextgen}
-      - POSTGRES_USER=${NEXTGEN_DATABASE_USER:-nextgen}
-      - POSTGRES_DB=${NEXTGEN_DATABASE_NAME:-nextgen}
+      - POSTGRES_PASSWORD=${NEXTGEN_DATABASE_PSWD:-local-pswd}
+      - POSTGRES_USER=${NEXTGEN_DATABASE_USER:-omnivector}
+      - POSTGRES_DB=${NEXTGEN_DATABASE_NAME:-jobbergate}
```

Comment:
not sure if this is useful. Can't you just set dummy fixed params here?

Reply:
They're just default values.  There isn't anything sensitive here. What would be the use of dummy values?

Reply:
I mean set fixed params like:

```shell
POSTGRES_PASSWORD=dummy-password
POSTGRES_USER=omnivector
POSTGRES_DB=jobbergate
```

It will avoid to set unnecessary env vars. You can ask me "ok but what's the problem on three more env vars?". The problem is the application itself is already complex so any new piece we add and increment the complexity makes it harder and harder to maintain. Does it make sense?

Reply:
Having the defaults lets us override them with a `.env` file or your local environment.

Reply:
that's exactly my point: there's no need to let them to be overwritten. It just adds complexity. I think we could just set whatever dummy value comes in mind and boom, ready to go

Reply:
@dusktreader can you describe a scenario when you set these environment variables to override the defaults?

Reply:
Here's a couple of scenarios:
* Running multiple dbs simultaneously so you need to map different ports (for example, running test dbs for license-manager-backend, jobbergate-api, armada-api)
* Running multiple instances of the same product's databases at the same time. For instance, If I wanted to have 2 versions of jobbergate (maybe I'm debugging an issue on the last released version while also working on new features that require a schema change)
* Connecting to a database hosted on another server (say connecting a local jobbergate with no database changes to the jobbergate database running in staging)

Reply:
👍